.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```

Comment:
We don't really want all of these `.idea` files kicking about.

Could you add them to the `.gitignore` and make sure that they aren't in this review?